packages/ssr-web/app/utils/wallet-info.ts

```diff
@@ -0,0 +1,43 @@
+class WalletAccount {
+  constructor(
+    readonly address: string,
+    readonly name: string,
+    readonly imageBase64: string | undefined = undefined
+  ) {}
```

Comment:
What does it mean to have arguments but not assign them? 😯

Reply:
Typescript magic for assigning properties from arguments, replacing
```js
class Something {
  readonly property: number;
  constructor(property: number) { this.property = number }
}
```

https://www.typescriptlang.org/docs/handbook/2/classes.html#parameter-properties

Reply:
oh nice 🎉